tensorflow/inference/buildspec-2-10.yml

```diff
@@ -3,6 +3,7 @@ region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK tensorflow
 version: &VERSION 2.10.1
 short_version: &SHORT_VERSION "2.10"
+device_type: &DEVICE_TYPE neuronx
```

Comment:
revert this change, this file does not concern with `neuron`

Reply:
Reverted